app/src/main/res/values/strings.xml

```diff
@@ -659,10 +659,7 @@
     <string name="backup_email_subject_2">%s database backup</string>
     <string name="backup_email_message_2">Your %s database containing bookmarks, notes, reading plans and workspaces is attached</string>
     <string name="restore_unsuccessfull">Restoring database was unsuccessful. Perhaps you gave invalid database file.</string>
-    <string name="backup_phone_storage">Phone storage</string>
-    <string name="backup_backup_title">Backup to where?</string>
-    <string name="backup_backup_message">Backup to phone or elsewhere via Share function (email, Google Drive etc.)?</string>
-    <string name="backup_share">Share</string>
+    <string name="backup_share">Backup</string>
```

Comment:
Need to check if this is old string in current stable version. We should not change old strings to completely different but instead create a new one with another id.

Reply:
yep backup_share is in current stable. So maybe rename to something else?

Reply:
Fixed now. We had backup_button already which is right context and string, so I just removed backup_share and used backup_button